start/manage/fleets/addShip.php

```diff
@@ -0,0 +1,100 @@
+<?php
+require '../ships/header.php';
+$id = isset($_GET['id']) ? $_GET['id'] : null;
+$errors = [];
+
+use Service\Container;
+use Model\ShipFleet;
+
+$container = new Container($configuration);
+
+$fleetLoader = $container->getFleetLoader();
+$shipLoader = $container->getShipLoader();
+$fleet = $fleetLoader->getFleetById($id);
+$fleetShips = $fleetLoader->getFleetShipsByFleet($fleet);
```

Comment:
You already have the Fleet object, so if you need the array of ShipFleets, you can just call `$fleet->getShipFleets()`. But I'm not sure that you need this anyway.